src/views/app/detail-panel/preview/mail-preview.jsx

```diff
@@ -528,7 +511,7 @@ export default function MailPreview({ message, expanded, isAlone, isMessageView
 					disableTransition
 					data-testid="MailMessageRendererCollapse"
 				>
-					{messageStatus === 'complete' && collapsedContent}
+					{message.isComplete && collapsedContent}
 				</Collapse>
 			</Container>
 		</Container>
```

Comment:
createSelector, selectMessages, selectMessagesStatus, getMsg, timezone, createSnackbar are not used anymore, I think they can be safely removed